src/javascript/app/Stores/Modules/SmartChart/smart-chart-store.js

```diff
@@ -63,6 +64,7 @@ export default class SmartChartStore extends BaseStore {
         this.setContractMode(false);
```

Comment:
@zakibinary can you please move this to when imported layout is done.

Reply:
I tried it like that initially, but it seems to cause a lag of disabling contract-mode when closing the contract from chart. That issue should be handled here: #9  with using the loader via `chartListener` API.